django_messages/templates/django_messages/compose.html

```diff
@@ -8,7 +8,7 @@ <h1>{% trans "Compose Message"%}</h1>
 <table>
 {{ form.as_table }}
 </table>
-<input type="submit" value="{% trans "Send" %} &raquo;"/>
+<input type="submit" value='{% trans "Send" %} &raquo;'/>
```

Comment:
This change is not needed. We should keep double quotes for the html attribute.

Reply:
Sure. So then you would need to change the double quotes around Send to single. Then: 
`<input type="submit" value="{% trans 'Send' %} &raquo;"/>`
The problem is that in the current version the first quote before Send closes the one of the value attribute, which leads to some sort of chaoes with the remaining 2 quotes.

Reply:
In fact I think this only breaks Syntax-Highlighting in your editor. The inner part {% trans "Send" %} is evaluated by the Django Template-Engine and the outer part (HTML) which becomes value="Send" is evaluated by your browser. There should be no situation, where the quotes interfere with each other.

Reply:
Oops yeah, it makes a lot sense